src/graphql/__generated__/resolvers-types.ts

```diff
@@ -61,6 +61,22 @@ export type CreatePlayerUnitInput = {
   unitId: Scalars['Int'];
 };
 
+export type DiceRollInput = {
+  cipherText: Scalars['String'];
+  publicKey: DiceRollPublicKeyInput;
```

Comment:
Sounds good, I can give this a more generic name then. Didn't want to assume all public keys would be passed this way in case it was only the dice service.

Reply:
See [0ed6d03](https://github.com/trumpet-zk-ignite-1/mina-arena-server/pull/27/commits/0ed6d03fff99e46b79a2bc4c52718ea9e91eaeb3) where I change this type name.